### PR TITLE
[Feature] GitHub webhook initialization

### PR DESCRIPTION
**Pull request by Sy1v4in**
This is the initialization of a webapp which can be plugged as a Github webhook through the `/github/webhook` route to log the posted informations.

### CODE REVIEW

**Comment by rehia** [comment]
I would recommend to use the template 
`<task|feat|docs|refactor|test|chore|perf|style>(<topic|context>): <message>`

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```
Comment (rehia) [inline build.sh]: What about making a `Makefile` ? :thinking:

**Review by rehia — COMMENTED** [review]
Let me know abut the `Makefile`

File: build.sh (hunk 1 above)
Comment (Sy1v4in) [reply build.sh]: First of all, I would say that these files are generated by the dotnet template; I didn't write it 😉 

Then, why not using a Makefile, right?
To me the main advantage to use a Makefile as:
```
target: dependencies
    command
```
is that the target will be built incrementally when its dependencies have changed.
But, my knowledges of the `dotnet` ecosystem is too poor to write a Makefile: I don't really know what produces and what are the dependencies of a `dotnet restore` and a `dotnet build` commands 🤷‍♂️
But don't hesitate to make a concret proposition 😉

**Comment by Sy1v4in** [comment]
For the commit messages, I agree. I would just want to replace the commit types by their emoji equivalents described as [gitmoji](https://gitmoji.carloscuesta.me/).
It is more fun don't you think? 😜

File: build.sh (hunk 1 above)
Comment (rehia) [reply build.sh]: From what I see, we need 2 targets
I'd suggest:

```suggestion
restore: ## Restore project dependencies
    dotnet restore src/Releasy

build: restore ## Build the whole project
    dotnet build src/Releasy

help:
    @grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'

.DEFAULT_GOAL := help
.PHONY: restore build help
```

**Comment by benjamingarcia** [comment]
> I would recommend to use the template
> `<task|feat|docs|refactor|test|chore|perf|style>(<topic|context>): <message>`

it exist another list with `ci|build`instead of `chore`

File: build.sh (hunk 1 above)
Comment (benjamingarcia) [reply build.sh]: If you need build, can I test gradle on it?

**Comment by Sy1v4in** [comment]
You're right @benjamingarcia we can improve the fist list provided by @rehia 
But, there are more than 60 gitmoji. So I think we can find our heart's desire 😄😍

File: build.sh (hunk 1 above)
Comment (Sy1v4in) [reply build.sh]: Of course, everybody can test what he or she wants 😉 And if the experiment is conclusive, why not to integrate it.
But, for now, I'm not sure I want to depend upon a new external tool. Moreover, I would like to be close enough to what is used in the Dockerfile.
And don't forget that, at this stage of the project, this build command run in about 5 sec. I'm not sure in the main painful point 💁‍♂️

Comment (Sy1v4in) [reply build.sh]: Fix in 851ae2c746bf54a2af4425bf8eb4450e656b6e84

**Review by rehia — COMMENTED** [review]
This feature looks totally awesome !!!

**Review by rehia — APPROVED** [review]

**Pull request merged**